src/docs/sphinx/dev_guide/updating_tpls.rst

```diff
@@ -189,24 +194,24 @@ More info on :ref:`building-axom-label`.
 Built-in TPLs
 -------------
 
-Axom several lightweight header-only libraries that we use internally and
-expose for downstream customers to use if they wish.
+Axom uses several, lightweight, header-only libraries internally which are
+exposed for downstream customers to use if they wish.
```

Comment:
```suggestion
Axom uses several lightweight, header-only libraries internally, which are
exposed for downstream customers to use if they wish.
```

I think this is more correct. maybe... the comma after several weirds me out but Word doesn't complain about it.